e2e/tests/nightly/registration.spec.ts

```diff
@@ -28,24 +29,39 @@ describe('User registration tests:', () => {
     await nextButton.clickWithEval();
 
     // Step 3: Enter user information
-    await createAccountPage.fillOutUserInformation();
+    const userId = await createAccountPage.fillOutUserInformation();
     nextButton = await createAccountPage.getNextButton();
     await nextButton.waitUntilEnabled();
     await nextButton.click();
 
     // Step 4: Enter demographic survey (All Survey Fields are optional)
     await createAccountPage.fillOutDemographicSurvey();
 
-    // TODO uncomment after disable recaptcha
```

Comment:
Here ^.  
`Submit` button click and verification were commented out before due to captcha verification initially. Sometimes later, Neha added `captchaSiteKey ` so captcha verification always pass. 
https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do

Reply:
test was not disabled or skipped, only the final step (Submit) were commented out to avoid failure.